tests/utils.py

```diff
@@ -31,3 +31,7 @@ def wrapper(test, *args, **kwargs):
         conf.overwrite()
         return test(*args, **kwargs)
     return decorator.decorator(wrapper, test)
+
+
+def output_equals(actual, expected):
```

Comment:
* The method doesn't do what it claims. We can call it `assert_cli_output` and document the expected behavior.
* It is better to combine assert and output_equals into a single method so you can output the actual and expected if they don't match. Currently, you only get a boolean output.